src/components/d3/createMapChart.js

```diff
@@ -2,39 +2,52 @@ import * as d3 from 'd3'
 import { adjustViewport } from '../../util/chart'
 import { margin } from '../../constants/chartConstants'
 
-const createMapChart = ({ data, width, height, el }) => {
+const createMapChart = ({ data, width, height, el, tip }) => {
   const [aWidth, aHeight] = adjustViewport(width, height, margin)
 
   const svg = d3.select(el).append('svg')
     .attr('width', aWidth)
     .attr('height', aHeight)
 
   const mapData = data.mapChartData
-  const heatmapStates = data.heatmapData.states
-  for (let i = 0; i < mapData.features.length; i++) {
-    mapData.features[i].properties.population = heatmapStates[i].population
-  }
+  const heatmapPopulation = data.heatmapData.countries
+  mapData.features.forEach(country => {
+    const nameOfCountry = country.properties.name
+    const countryPopulation = heatmapPopulation.find(({ name }) => nameOfCountry === name)
+    if (countryPopulation) {
+      country.properties.population = countryPopulation.population
+    } else {
+      country.properties.population = 0
+    }
+    return country
+  })
 
   const projection = d3
-    .geoAlbersUsa()
-    .fitExtent([[0, 0], [aWidth, aHeight]], mapData)
+    .geoMercator()
+    .fitExtent([[0, 0], [aWidth, aHeight * 1.5]], mapData)
```

Comment:
Had to do this to hide Antartica (otherwise the map is scaled oddly...)

Reply:
Okay, I'm going to merge this in now